superset/connectors/sqla/views.py

```diff
@@ -121,6 +128,24 @@ class TableColumnInlineView(CompactCRUDMixin, SupersetModelView):  # noqa
         "python_date_format": _("Datetime Format"),
         "type": _("Type"),
     }
+    validators_columns = {
+        "python_date_format": [
+            # Restrict viable values to epoch_s, epoch_ms, or a strftime format
+            # which adhere's to the ISO 8601 format (without time zone).
+            Regexp(
```

Comment:
@dpgaspar this FAB validator works but it doesn't seem to show the message. I've successfully used validators in Superset on other PRs and I was wondering whether there's a FAB issue displaying messages when the form field isn't on the primary tab.